resources/i18n/fr.json

```diff
@@ -0,0 +1,108 @@
+{
+    "@metadata": {
+		"authors": [
+		]
+	},
+    "name":"Français",
+    "error-downloader-window-title":"Impossible de lancer le system de téléchargement",
+    "error-downloader-launch-message":"Impossible de lancer le system de téléchargement, Kiwix-desktop fonctionnera mais sans les fonctionnalités de téléchargement !",
+    "open-zim":"Ouvrir un Zim",
+    "local-kiwix-server":"Serveur local Kiwix",
+    "random-article":"Article au hasard",
+    "home-page":"Page principale",
+    "print":"Imprimer",
+    "new-tab":"Nouvel onglet",
+    "close-tab":"Fermer l'onglet",
+    "reopen-closed-tab":"Rouvrir l'onglet fermé",
+    "browse-library":"Parcourir la bibliothèque",
+    "open-file":"Ouvrir un fichier",
+    "open-recent":"Fichiers récemment ouverts",
+    "search-article":"Chercher un article",
+    "search-in-library":"Chercher dans la bibliothèque",
+    "find-in-page":"Chercher dans la page",
+    "set-fullscreen":"Mode plein écran",
+    "quit-fullscreen":"Quitter le plein écran",
```

Comment:
Maybe `Quitter le mode plein écran` ?

Reply:
I've use the translation in previous translation file :

https://github.com/kiwix/kiwix-desktop/commit/ffdfff514e169eea42ba7b3d86b48f01120c505e#diff-85adf1c107d0d00b2b7d1570fafc5a70L106